homeassistant/components/netgear_lte/strings.json

```diff
@@ -80,5 +80,59 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "mobile_connected": {
+        "name": "Mobile connected"
+      },
+      "roaming": {
+        "name": "Roaming"
+      },
+      "wire_connected": {
+        "name": "Wire connected"
+      }
+    },
+    "sensor": {
+      "band": {
+        "name": "Current band"
+      },
+      "cell_id": {
+        "name": "Cell ID"
+      },
+      "connection_text": {
+        "name": "Connection text"
+      },
+      "connection_type": {
+        "name": "Connection type"
+      },
+      "radio_quality": {
+        "name": "Radio quality"
+      },
+      "register_network_display": {
+        "name": "Register network display"
+      },
+      "rx_level": {
+        "name": "Rx level"
+      },
+      "service_type": {
+        "name": "Current PS service type"
```

Comment:
What's a PS?

Reply:
The sensor for mine has a state of LTE. Per the following, I assume it is Packet Switching: https://mobilepacketcore.com/mobile-network-technologies/

Reply:
might be best to have it simple: "Service type".

Reply:
I mean if we don't have a clear definition (that's not an assumption), why would we put it in the entity name? :P

Reply:
Sounds good!